course_content/notebooks/numpy_intro.ipynb

```diff
@@ -1123,21 +965,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print('unmasked average = ', np.mean(daily_records))\n",
-    "print('masked average = ', np.ma.mean(masked_data))"
+    "print('Unmasked average: {}'.format(np.mean(data)))\n",
+    "print('Masked average: {}'.format(np.ma.mean(masked_data)))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `np.ma.masked_array()` function seen above is a simple creation method for masked data.  \n",
-    "The sub-module `np.ma` contains all the NumPy masked-data routines.\n",
-    "\n",
-    "Instead of masking selected points, as above, a mask is often specified as a \"mask array\":  This is a boolean array of the same size and shape as the data, where `False` means a good datapoint and `True` means a masked or missing point.  Such a 'boolean mask' can be passed in the `np.ma.masked_array` creation method, and can be extracted with the `np.ma.getmaskarray()` function.\n",
-    "\n",
-    "Note that most file storage formats represent missing data in a _different_ way, using a distinct \"missing data\" value appearing in the data.  \n",
-    "There is special support for converting between this type of representation and NumPy masked arrays :  Every masked array has a 'fill_value' property and a 'filled()' method to convert between the two."
+    "Note that most file formats represent missing data in a _different_ way, using a distinct \"missing data\" value appearing in the data. There is special support for converting between this type of representation and NumPy masked arrays. Every masked array has a 'fill_value' property and a 'filled()' method to convert between the two."
```

Comment:
`filled()` doesn't convert *between* `--` and the `fill_value` but converts `--` *to* the `fill_value`.
i.e. `filled()` only functions in one direction and does not also convert an array with filled values back to a masked array with `--` values.